src/RNTupleReader.cc

```diff
@@ -141,15 +141,10 @@ std::unique_ptr<ROOTFrameData> RNTupleReader::readEntry(const std::string& categ
   auto dentry = m_readers[category][0]->GetModel()->GetDefaultEntry();
 
   for (size_t i = 0; i < m_collectionInfo[category].id.size(); ++i) {
-    const auto collectionClass = TClass::GetClass(m_collectionInfo[category].type[i].c_str());
-
-    auto collection =
-        std::unique_ptr<podio::CollectionBase>(static_cast<podio::CollectionBase*>(collectionClass->New()));
```

Comment:
Was this line problematic?

Reply:
No, not really per se. It has just become unnecessary with the introduction of our own buffer factory. Now it was only used to get the type name basically, which we already have available otherwise.